### ISSUE TITLE
Error due to type conflict in new version of tibble

### ISSUE THREAD

**Issue by jaspercooper** [jessecambon/tidygeocoder#49]
hi @jessecambon — thanks for this awesome package. It solves a huge problem in my workflow!

I've encountered a hiccup that appears to be due to tibble. The following code ran fine before I updated tibble, but broke afterward:

```
geo(
  street = "9500 Gilman Dr.,",
  city = "San Diego",
  state = "CA",
  postalcode = "92161",
  method = "cascade"
)
```

The error reads:

```
Error: Assigned data `retry_results` must be compatible with existing data.
ℹ Error occurred for column `lat`.
✖ Can't convert from <double> to <logical> due to loss of precision.
* Locations: 1.
```

The error seems to be coming from here: https://github.com/jessecambon/tidygeocoder/blob/d971ebc4b420eb3b8a1b8cacc2d7960c2a12a81b/R/geo_methods.R#L70

Tibble is not happy that the variable types are being overwritten. 

If instead you change line 70 to something more explicit like `combi[,c("lat","long")] <- retry_results[,c("lat","long")]` everything runs just fine. 

Hope this helps, thanks again for the great package!

**Comment by jessecambon** [comment]
Thanks for reporting this, I was able to replicate the issue. It looks like this error is coming from vctrs:

https://vctrs.r-lib.org/reference/vec_cast.html

I haven't gotten a chance to dig into it yet, but the solution may be to force the lat/long coordinates to be formatted as a double in the [extract_results](https://github.com/jessecambon/tidygeocoder/blob/master/R/utils.R#L52) or geo functions.

Using the Census geocoder for this address is returning 'logical' format for the lat/long columns currently:

```
a <- geo(
    street = "9500 Gilman Dr.,",
    city = "San Diego",
    state = "CA",
    postalcode = "92161",
    method = "census"
)

summary(a)
#>    street              city              state            postalcode          lat            long        
#> Length:1           Length:1           Length:1           Length:1           Mode:logical   Mode:logical  
#> Class :character   Class :character   Class :character   Class :character   NA's:1         NA's:1        
#> Mode  :character   Mode  :character   Mode  :character   Mode  :character   
```

When using cascade, the numeric results of the 'osm' geocoder are being put into these 'logical' columns which causes the error.

**Comment by jaspercooper** [comment]
Yeah, that sounds right. It may be as simple as fixing the variable types in the `initial_results` tibble. I'd check out `debugonce(geo_cascade)` again though --- it also appears the subsetting by `na_indices` around line 70 isn't quite doing what it's supposed to (presumably singling out the columns with NAs), as it's defined as a scalar on line 56.

**Comment by jessecambon** [comment]
@jaspercooper I've got a bug fix for you to try out:

```
devtools::install_github("jessecambon/tidygeocoder", ref = 'cascade-na-fix')
```

I changed the the lat/long NA values from `NA` to `as.numeric(NA)` in the `extract_results()` and `get_na_results()` functions in utils.R. I haven't tested too extensively yet so let me know if you run into any issues. The `na_indices` variable in geo_method.R is for the row numbers for addresses that were not found by the first method tried.
